test/jellyfish_web/integration/server_notification_test.exs

```diff
@@ -35,6 +41,15 @@ defmodule JellyfishWeb.Integration.ServerNotificationTest do
 
   @source_uri "rtsp://placeholder-19inrifjbsjb.it:12345/afwefae"
 
+  @files ["manifest.m3u8", "header.mp4", "segment_1.m3u8", "segment_2.m3u8"]
+  @body <<1, 2, 3, 4>>
```

Comment:
Remove this attribute, also in `component_controller_test`

Reply:
I cannot do that. Component controller tests if hls manager sends correct data to s3